plugins/repository-s3/docker-compose.yml

```diff
@@ -2,8 +2,8 @@ version: '3'
 services:
   minio-fixture:
     build:
-      context: ./build/minio-docker
+      context: ./testfixtures_shared/minio-docker
```

Comment:
This is just the directory used for building the image, right? This folder isn't used at runtime. Since this is "build" output I think we can continue to keep this under the `build` directory so it gets cleaned up by the `clean` task.